Data/SourceAssets/Localization/English/Items.txt

```diff
@@ -127,27 +127,27 @@
 
 
 
-(Das ist unverk�uflich
-(Ein Pfeil kostet = Goldst�cke
-(Die Feuerb�lle kosten = Goldst�cke
-(Ein Stein kostet = Goldst�cke
-(Die Medizin kostet = Goldst�cke
-(Der Dolch kostet = Goldst�cke
-(Das Schwert kostet = Goldst�cke
-(Die Armbrust kostet = Goldst�cke
-(Die Rattenfalle kostet = Goldst�cke
-(Die Konduktorkugel kostet = Goldst�cke
-(Die Schaufel kostet = Goldst�cke
-(Das Pfeilgift kostet = Goldst�cke
-(Folie
+(This is not for sale.
+(One arrow costs  = Gold coins
+(The fireballs cost    = Gold coins
+(One stone costs  = Gold coins
+(The medicine costs = Gold coins
+(The blade costs  = Gold coins
+(The sword costs    = Gold coins

```

Comment:
why the whitespaces before the = ? Since we are packing the files again, the lengths do not have to match, unless for on-screen formatting reasons.

Reply:
No clue, where these came from. Thanks for all the corrections. Applied as suggested